### PR TITLE
Update SivaReader.java

### PR DESCRIPTION
**Pull request by berry120**
Enable SivaReader to use the AutoCloseable interface.

### CODE REVIEW

**Review by erizocosmico — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -52,15 +52,14 @@ public class Main {
     public static void main(String[] args) {
         try {
             LOGGER.log(Level.INFO, "unpacking siva-file");
-            SivaReader sivaReader = new SivaReader(new File(DEFAULT_SIVA_FILE));
-            List<IndexEntry> index = sivaReader.getIndex().getFilteredIndex().getEntries();
-            for (IndexEntry indexEntry : index) {
-                InputStream entry = sivaReader.getEntry(indexEntry);
-                Path outPath = Paths.get(SIVA_UNPACKED_DIR.concat(indexEntry.getName()));
-                FileUtils.copyInputStreamToFile(entry, new File(outPath.toString()));
+            try (SivaReader sivaReader = new SivaReader(new File(DEFAULT_SIVA_FILE))) {
```
Comment (mcarmonaa) [inline README.md]: Could we avoid to have a nested `try` using just this `try` catching the exception?

File: README.md [hunk 2]
```diff
@@ -50,17 +50,14 @@ public class Main {
     private static final Logger LOGGER = Logger.getLogger(Main.class.getName());
 
     public static void main(String[] args) {
-        try {
+        try (SivaReader sivaReader = new SivaReader(new File(DEFAULT_SIVA_FILE))) {
```
Comment (mcarmonaa) [inline README.md:54]: sorry @berry120 , could you get out of the `try` block the logging line `LOGGER.log(Level.INFO, "unpacking siva-file")` so this is logged although the siva reader opening fails?

Thanks for your contribution!

File: README.md (hunk 1 above)
Comment (berry120) [reply README.md]: Very good point. No idea why I didn't just remove the nested try initially. Should now be resolved.

File: README.md (hunk 2 above)
Comment (berry120) [reply README.md:54]: Yup, another good point! That's now done too.

**Review by mcarmonaa — APPROVED** [review]

**Comment by berry120** [comment]
Anyone able to re-run the Travis check? Looks like it may just be a transient issue (I can't think why anything I've changed would affect it on just one version of the JDK.)

**Pull request merged**